AzureDevOpsPowerShellAPI/Public/Api/Policy/Configuration/Set-AzDoBranchPolicyCommentResolution.ps1

```diff
@@ -33,23 +33,18 @@ function Set-AzDoBranchPolicyCommentResolution {
   [CmdletBinding(SupportsShouldProcess)]

```

Comment:
Since it's a potential invasive procedure and we have a should process block
```suggestion
  [CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'High')]
```